LoopKitUI/Views/Settings Editors/TherapySettingsView.swift

```diff
@@ -418,27 +347,26 @@ struct SectionWithEdit<Content, NavigationDestination>: View where Content: View
                 Text(title)
                     .bold()
                 Spacer()
-                DescriptiveText(label: descriptiveText)
+                ZStack(alignment: .leading) {
+                    DescriptiveText(label: descriptiveText)
+                    NavigationLink(destination: destination, isActive: $activate) {
+                        EmptyView()
+                    }
+                }
                 Spacer()
             }
             content()
-            if isEditing {
-                navigationButton
-            }
         }
+        .highPriorityGesture(
```

Comment:
Just curious, does the gesture only work if made high priority?

Reply:
I *thought* you'd notice that :). The gesture gets handled by the "deepest view" first, and then propagates up.  In this case, for example, I'm (re-)using some editors that want to claim the tap for themselves.  Since the intent is to navigate "no matter what", I thought that an overriding "highPriority" handler was appropriate.

Reply:
Ah, forgot those cells have a gesture recognizer attached already. Makes sense!